modules/apps/account/account-admin-web/package.json

```diff
@@ -1,14 +1,16 @@
 {
 	"dependencies": {
 		"frontend-js-web": "*",
-		"metal-dom": "2.16.8",
-		"metal-events": "2.16.8"
+		"metal-path-parser": "1.0.4",
+		"metal-router": "3.6.3",
+		"metal-state": "2.16.8",
+		"senna": "2.7.9"
 	},
-	"name": "account-admin-web",
+	"name": "portal-portlet-bridge-soy-impl",
 	"scripts": {
 		"build": "liferay-npm-scripts build",
 		"checkFormat": "liferay-npm-scripts check",
 		"format": "liferay-npm-scripts fix"
 	},
-	"version": "2.0.0"
+	"version": "4.0.0"
```

Comment:
Why this version change?

Reply:
Same as above, I discarded all my changes but for some reason that remained.